config/link-check/config.yml

```diff
@@ -0,0 +1,12 @@
+rootURL: https://dvc.org
```

Comment:
Does this rootURL need to be changed to mlem.ai?

Reply:
Only for the daily check, but good point nonetheless!